mlrun/utils/helpers.py

```diff
@@ -47,6 +47,10 @@
 hub_prefix = "hub://"
 DB_SCHEMA = "store"
 
+legal_time_units = ["year", "month", "day", "hour", "minute", "second"]
+default_time_partitions = ["year", "month", "day", "hour"]
+default_time_partitioning_granularity = "hour"
```

Comment:
```suggestion
LEGAL_TIME_UNITS = ["year", "month", "day", "hour", "minute", "second"]
DEFAULT_TIME_PARTITIONS = ["year", "month", "day", "hour"]
DEFAULT_TIME_PARTITIONING_GRANULARITY = "hour"
```
Lets try to keep to the convention that params in global scope will be CAPS.